src/get-started/labels-legends-headings/index.md.njk

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout-pane.njk
+title: Making labels and legends headings
+section: Get started
+theme: How to guides
+order: 4
+description: How to markup and style labels, legends and headings
+---
+{% from "_example.njk" import example %}
+
+Whenever you design a form, you should by splitting it across multiple pages with each page containing just one thing, for example:
```

Comment:
This part of the sentence doesn't make sense:

> you should by splitting it across multiple pages with each page containing just one thing

– you should _what_?

Reply:
(Also, we tell users to 'start' with one thing per page, but that they might merge pages together. So should this make it clearer that this only applies when a page does actually only have one 'question' on it?)

Reply:
This should say "you should _start_ by splitting it across multiple pages with each page containing just one thing." -  I copied this straight from the Service Manual so it must have been missed from their content. I will let them know.

And OK let's just remove the bit about merging pages from here - it's not relevant to this guidance, and we link out to the more detailed explanation of one thing per page anyway.